webapp/client/src/pages/InfoTaxReturnForPensionersPage.js

```diff
@@ -0,0 +1,255 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import { Trans, useTranslation } from "react-i18next";
+import AnchorButton from "../components/AnchorButton";
+
+const QuestionBox = styled.div`
+  padding: 50px 0;
+`;
+
+const QuestionBoxBackground = styled.div`
+  background: var(--bg-highlight-color);
+  height: 459px;
+  width: calc(100vw - 220px);
+  position: relative;
+  left: calc(-50vw + 82%);
+
+  display: flex;
+  justify-content: center;
+
+  @media (max-width: 1042px) {
+    left: calc(-50vw + 85.2%);
+  }
+
+  @media (max-width: 1024px) {
+    // left: calc(-50vw + 85%);
+    left: -1.75rem;
+    width: 100vw;
+  }
+
+  @media (max-width: 900px) {
+    left: -1rem;
+    width: 100vw;
+  }
+
+  @media (max-width: 360px) {
+    height: 615px;
+  }
+`;
+
+const HeadingText = styled.h1`
+  font-size: 2.25rem;
+  @media (max-width: 320px) {
+    font-size: 2.25rem;
+  }
+`;
+const ParagraphHeadingText = styled.p`
+  font-size: 1.75rem;
+  @media (max-width: 320px) {
+    font-size: 1.5rem;
+  }
+`;
+const ParagraphTextLarger = styled.p`
+  font-size: 1.5rem;
+
+  @media (max-width: 320px) {
+    font-size: 1.25rem;
+  }
+`;
+
+const ParagraphTextMedium = styled.p`
+  font-size: 1.25rem;
+`;
+
+const TestBack = styled.div`
+  padding-left: var(--spacing-03);
+  padding-right: var(--spacing-03);
+  margin: auto;
+  flex: 1;
+  width: 100%;
+  max-width: var(--main-max-width);
+`;
+
+const ContentTopSpacing = styled.div`
+  margin-top: 9.375rem;
+  position: relative;
+
+  & img {
+    width: 100%;
+  }
+`;
+
+const QuestionBoxAnchorButtons = styled.div`
+  @media (max-width: 605px) {
+    display: flex;
+    flex-direction: column;
+  }
+`;
+
+const ListBox = styled.div`
+  background: var(--bg-highlight-color);
+  padding: 32px;
+  list-style-type: none;
+
+  @media (max-width: 320px) {
+    padding: 16px;
+  }
+`;
```

Comment:
Personally, I think a lot of this stuff could be its own component because we might use it more than once, correct?

Reply:
Yes agreed. The initial thought was to create a separate component for the "QuestionBox" at the bottom of the page thinking that we were going to use it for other possible pages. But we went with keeping everything confined to this single test page for now.